learn/advanced/updating.md

```diff
@@ -253,6 +253,10 @@ Now that you’ve got your dump, [install the latest version of MeiliSearch](/le
 ./meilisearch --import-dump /dumps/your_dump_file.dump
 ```
 
+::: warning
+If you are using MeiliSearch v0.20.0 or below, you will need to import your dump to a version between v0.21.0 and v0.25.0 and the re-export it to the newer version.
```

Comment:
What do you think about this? Too wordy?

I think the important thing to keep in mind here is when a dump must be exported and when it must be imported.

```suggestion
If you are using MeiliSearch v0.20 or below, migration should be done in two steps. First, import your v0.20 dump into an instance running any version of MeiliSearch between v0.21 and v0.25. Second, export another dump from this instance and import it to a final instance running your targeted version.
```